graphbolt/CMakeLists.txt

```diff
@@ -178,30 +221,40 @@ if(USE_CUDA)
   target_link_libraries(${LIB_GRAPHBOLT_NAME} ${LIB_GRAPHBOLT_CUDA_NAME})
 
 elseif(USE_HIP)
-  file(GLOB BOLT_CUDA_EXTENSION_SRC
+  file(GLOB LIB_GRAPHBOLT_HIP_SRC
```

Comment:
I think better to keep the same naming convention as for cuda, so `BOLT_HIP_EXTENSION_SRC`. Though actually, I think from a maintenance perspective it would be better to avoid duplicating the variable (and anything else that's the same) unless and until they diverge:

```cmake
if(USE_CUDA OR USE_HIP)
  file(GLOB BOLT_CUDA_EXTENSION_SRC
    ${BOLT_DIR}/cuda/extension/*.cu
    ${BOLT_DIR}/cuda/extension/*.cc
    ../third_party/HugeCTR/gpu_cache/src/nv_gpu_cache.cu
  )
  # Anything else common between the two
  if(USE_CUDA)
    # ...
  elseif(USE_HIP)
    # ...
  endif()
endif()
```

Reply:
Should be addressed in https://github.com/ROCm/dgl/pull/3/commits/c95b33d1ccdd28d891495502ca4a0ca587c132d5, lmk what you think